tests/models/encoder_decoders/wikitables_semantic_parser_test.py

```diff
@@ -25,6 +25,18 @@ def setUp(self):
     def test_model_can_train_save_and_load(self):
         self.ensure_model_can_train_save_and_load(self.param_file)
 
+    def test_average_encoder_works_simple(self):
```

Comment:
Because the function you're testing here is going away with a switch to `TimeDistributed`, you can remove this test.  But it'd be good to add tests for `_get_neighbor_indices` and `_get_type_vector`.